native/src/routes/SearchModal.tsx

```diff
@@ -102,6 +102,7 @@ const SearchModal = ({
       contentWithoutHtml={item.contentWithoutHtml}
       language={languageCode}
       query={query}
+      accessibilityHint={t(`It navigates to the description of the item`)}
```

Comment:
This should be moved to our translations.json file, so the users get this output in their language.
Also i think "description" is not really the correct term, maybe "page" is better.
Better place to put this, is in the `SearchListItem.tsx` as all searchListItems have the same hint, no need to pass it around with props.

Reply:
@ztefanie if I put it inside `SearchListItem` it causes error

Reply:
Not sure which error you get, but here is the solution i suggest: https://github.com/digitalfabrik/integreat-app/pull/2583/files

Reply:
@ztefanie I also did the same thing, let me try it again :)